list

```diff
@@ -1331,6 +1333,7 @@ urlify.cn
 urlr.me
 urls.fr
 urls.kr
+urlshrt.co
```

Comment:
From the example https://urlshrt.co/ElDM, it looks a bit different from other URL shortener behavior, it doesn't redirect directly, but displays some advertisements, which I may need to dig deeper into.

Reply:
Let me know when it's removed, then, thanks!